configuration/NI.Analysis.targets

```diff
@@ -0,0 +1,107 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <!-- TODO: This location of 'content' needs to get updated once we build our package if we decide to put the files elsewhere, outside of a 'content' directory -->
```

Comment:
I added this TODO to note that we may need to update this once we create our new package. If it stays the same, we should just remove the TODO.

Reply:
Do we expect clients to define `PkgNI_DotNet_Analyzers`? If so, I'd just get rid of "content"